derive/tests/implicit.pest

```diff
@@ -1,9 +1,9 @@
 program = _{ SOI ~ implicit ~ EOI  }
-implicit= ${ or ~ (WHITESPACE+ ~ or )* }
+implicit= ${ #head = or ~ #tail = (WHITESPACE+ ~ or)* }
 
-or  = !{ and ~ (or_op ~ and)+ | and }
-and = { comp ~ (and_op ~ comp)+ | comp }
-comp = { array ~ eq_op ~ array | array }
+or  = !{ #more_and = and ~ (or_op ~ and)+ | #one_and = and }
```

Comment:
The `#more_and` here only grabs the `and`, not all of `and ~ (or_op ~ and)+`, right?

Reply:
yes, I think so, it'd need extra brackets to capture the full expression